### PR TITLE
Balance Pass hak changes

### PR DESCRIPTION
**Pull request by JohnWildkins**
see Zunath/SWLOR_NWN#1487

Adds definitions for:
 - Flurry Style and Mastery
 - Crushing Style and Mastery
 - Duelist
 - Shield Master
 - Wailing Blows
 - Dirty Blow
 - Shield Bash I, II, III
 - Force Inspiration I, II, III
 - Bulwark

### CODE REVIEW

**Comment by zunath** [comment]
Merge conflict on this one. Likely due to another PR being merged in.

**Comment by JohnWildkins** [comment]
forgot to actually commit the baseitems.2da change that sets everything's base crit range to 20 (5%)

**Review by zunath — CHANGES_REQUESTED** [review]
Just need confirmation on the baseitems.2da file. Let me know, looks good otherwise.

File: swlor2_2da/baseitems.2da [hunk 1]
```diff
@@ -13,8 +13,8 @@
...
+12    172        twobladedsword          2              5               0x1C010          1               2           WDbSw         0                1             1             1             it_bag          iwdbsw            0           8             3            4            ****           1.5              10         100        1         8           1            2             1          1          1          2                16860217      10             8          0          0            1            44         ****       ****       ****       ****       4            0        0               5431              0                 1                150        11              ****             ****          ****                 50            50            50            99               1              127               655                   165                        693                            89                           745                                  531                                 943                  ****           ****
```
[the remark this answers is not in the document]
Comment (zunath) [reply swlor2_2da/baseitems.2da:16]: Well, just saw your comment above about the crit ranges. Sorry about that. This is fine to merge in. I will hold off on actually merging it until the changes in SWLOR_NWN are finished up.

**Review by zunath — APPROVED** [review]

**Comment by zunath** [comment]
Holding on merging until changes in SWLOR_NWN are ready.

**Pull request merged**